com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- The faceset names were being read in the incorect order in certain cases.
```

Comment:
Suggestion:
`Make sure Unity always reads the Face Set names in the correct order.`